src/components/Menu/ActionEntry.scss

```diff
@@ -1,16 +1,9 @@
 %text__is-active-item-btn {
 	opacity: 1;
-
-	&::before {
-		transform: translateX(-50%);
-		border-radius: 100%;
-		position: absolute;
-		background: var(--color-primary-element);
-		bottom: 0;
-		height: 6px;
-		width: 6px;
-		content: '';
-		left: 50%;
+	background-color: var(--color-primary-light);
+	border-radius: 50%;
+	svg {
+		fill: var(--color-primary) !important;
```

Comment:
Is it possible to avoid `!important` here and in line 75?

Reply:
I tried to get rid of it but we need it in those lines.

Reply:
Maybe setting `color: var(--color-primary);` on the parent could work as the svg should use currentColor?